### PR TITLE
Javascript javascript1 week1

### PR DESCRIPTION
**Pull request by Vahablotfi**
this is the link for freecodecamp part of the Homework
https://www.freecodecamp.org/vahablotfi

### CODE REVIEW

File: git/week1/class_playground/apples_file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+kdhfiuwehfoielkhsdkgjgoiuwepojwelfhwljhi;l
```
Comment (aina21) [inline git/week1/class_playground/apples_file.txt]: unrelated file, It shouldn't be in this PR.

File: git/week1/class_playground/bananas_file.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+w;eHFJKDNSDLUYQGIWgit
```
Comment (aina21) [inline git/week1/class_playground/bananas_file.txt]: unrelated file, It shouldn't be in this PR.

File: git/week1/class_playground/oranges_file.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+p'oe;dklmsakjasq;iowldjksn;iuhwL;mlkjsloihwjkwdsagit addgikjbhwaSD.JHKas.kjasCKJ,
```
Comment (aina21) [inline git/week1/class_playground/oranges_file.txt]: unrelated file, It shouldn't be in this PR.

File: javascript/javascript1/week1/homework of  JSweek1/index.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+const yearOfBirth = 1991;
+const yearFuture = 2063;
+const age = yearFuture - yearOfBirth ;
+console.log("You will be " + age + " years old in " + yearFuture)
+
+//Goodboy-Oldboy(a DOG AGE CALCULATOR)
+const dogYearOfBirth = 2017;
+const  dogYearFuture = 2027;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = dogYearFuture - dogYearOfBirth;
```
Comment (aina21) [inline javascript/javascript1/week1/homework of  JSweek1/index.js:11]: why do you use the number for the shouldShowResultInDogYears variable?

File: javascript/javascript1/week1/homework of  JSweek1/index.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+if(shouldShowResultInDogYears != 70){
+    console.log("Your dog will be ",dogYear," human years old in",dogYearFuture,".")
+} else {
+    console.log("Your dog will be",dogYear,"dog years old in",dogYearFuture )
```
Comment (aina21) [inline javascript/javascript1/week1/homework of  JSweek1/index.js:15]: what is the difference between human years and dog years?

**Review by aina21 — CHANGES_REQUESTED** [review]
good job :+1:

File: javascript/javascript1/week1/homework of  JSweek1/index.js (hunk 4 above)
Comment (Vahablotfi) [reply javascript/javascript1/week1/homework of  JSweek1/index.js:11]: I'm sorry I don't get what you mean, but my I wrote it, so if you change the value of  dogYearOfBirth variable or  dogYearFuture  variable so it gives a different result for  shouldShowResultInDogYears variable.

File: javascript/javascript1/week1/homework of  JSweek1/index.js (hunk 5 above)
Comment (Vahablotfi) [reply javascript/javascript1/week1/homework of  JSweek1/index.js:15]: the dog year supposed to be70
